content/how-to/develop-using-vagrant.md

```diff
@@ -0,0 +1,92 @@
+---
+title: How to Develop using Vagrant
+---
+
+If you have a more advanced project and use [Vagrant](https://www.vagrantup.com/) to run your development environment in a Virtual Machine, you'll often want to also run webpack in the VM.
+
+To start, make sure that the `Vagrantfile` has a static IP;
```

Comment:
It feels like this would be a good place for a subtitle right after the intro. Consistency thing.